sdk/rdbms/azure-mgmt-rdbms/CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Release History
 
+## 1.10.0 (2020-02-21)
+
+**Features**
+
+- Added operation group PrivateLinkResourcesOperations
+- Added operation group PrivateEndpointConnectionsOperations
+- Added operation group AdvisorsOperations
+- Added operation group MySQLManagementClientOperationsMixin
+- Added operation group QueryTextsOperations
+- Added operation group RecommendedActionsOperations
+- Added operation group MariaDBManagementClientOperationsMixin
+- Added operation group LocationBasedRecommendedActionSessionsOperationStatusOperations
+- Added operation group LocationBasedRecommendedActionSessionsResultOperations
+- Added operation group WaitStatisticsOperations
+- Added operation group TopQueryStatisticsOperations
+
```

Comment:
we need a note about general breaking changes

Reply:
please help to check the change

Reply:
Change and please help to review